Networking/Networking/Remote/FeatureFlagRemote.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+
+/// Protocol for `FeatureFlagsRemote` mainly used for mocking.
+///
+public protocol FeatureFlagRemoteProtocol {
+    func loadAllFeatureFlags() async throws -> [RemoteFeatureFlag: Bool]
+}
+
+/// Feature Flags: Remote Endpoints
+///
+public class FeatureFlagRemote: Remote, FeatureFlagRemoteProtocol {
+    public func loadAllFeatureFlags() async throws -> [RemoteFeatureFlag: Bool] {
+        let parameters: [String: String] = [
+            ParameterKeys.platform: "ios",
+            ParameterKeys.marketingVersion: Bundle.main.marketingVersion,
+        ]
+
+        let request = DotcomRequest(wordpressApiVersion: .wpcomMark2, method: .get, path: Paths.lookup, parameters: parameters)
+        let valuesByFeatureFlagString: [String: Bool] = try await enqueue(request)
+        return Dictionary(uniqueKeysWithValues: valuesByFeatureFlagString.compactMap { key, value in
+            guard let featureFlag = RemoteFeatureFlag(rawValue: key) else {
+                return nil
+            }
+            return (featureFlag, value)
+        })
+    }
+}
+
+public enum RemoteFeatureFlag: Decodable {
+    case storeCreationCompleteNotification
+    case oneDayAfterStoreCreationNameWithoutFreeTrial
+    case oneDayBeforeFreeTrialExpiresNotification
+    case oneDayAfterFreeTrialExpiresNotification
+
+    init?(rawValue: String) {
+        switch rawValue {
+        case "woo_notification_store_creation_ready":
+            self = .storeCreationCompleteNotification
+        case "woo_notification_nudge_free_trial_after_1d":
+            self = .oneDayAfterStoreCreationNameWithoutFreeTrial
+        case "woo_notification_1d_before_free_trial_expires":
+            self = .oneDayBeforeFreeTrialExpiresNotification
+        case "woo_notification_1d_after_free_trial_expires":
+            self = .oneDayAfterFreeTrialExpiresNotification
+        default:
+            return nil
+        }
+    }
+}
+
+// MARK: - Constants!
+//
+private extension FeatureFlagRemote {
+    enum Paths {
+        static let lookup = "mobile/feature-flags"
```

Comment:
Nit: Is there a reason to name this as `lookup`?

Reply:
This is actually pre-existing code https://github.com/woocommerce/woocommerce-ios/blob/5ee53c9d9d10ad0c000715a91de7de998f4c1249/Networking/Networking/Remote/FeatureFlagsRemote.swift#L30 (shown as new code in GitHub because of the renaming), I thought I'd keep as much as the same code. I think it means API "GET"/"READ" here